spec/numberformat.html

```diff
@@ -175,7 +175,7 @@ <h1>PartitionNumberPattern ( _numberFormat_, _x_ )</h1>
         1. Let _exponent_ be 0.
         1. If _x_ is *NaN*, then
           1. Let _n_ be an implementation- and locale-dependent (ILD) String value indicating the *NaN* value.
-        1. Else if _x_ is not a finite Number or BigInt,
+        1. Else if _x_ is not a finite Number,
```

Comment:
"is a Number that is not finite", or "is a non-finite Number", because by one reading a BigInt is not a finite Number?